source/Glossary/throttling.md

```diff
@@ -1,37 +1,31 @@
 ---
-layout: page
-weight: 0
-title: Throttling
 seo:
-  title: Email Throttling
+  title: Throttling
+  description: Learn more about deferrals and throttling
+  keywords: deferred, deferral, throttling, bounce, block
+title: Throttling
+weight: 0
+layout: page
 navigation:
   show: false
 ---
 
-So, you tried to send a bunch of email, and when you checked your [Event webhook]({{root_url}}/API_Reference/Webhooks/event.html) or your [Email Activity]({{root_url}}/User_Guide/Delivery_Metrics/email_activity.html) page, you see a number of messages are marked as "Deferred" instead of "Delivered". Or maybe you're looking at your [Email Reports]({{root_url}}/User_Guide/Delivery_Metrics/email_reports.html), and noticing a lot of addresses in your [Blocks]({{site.app_url}}/suppressions/blocks), but not many in your Hard Bounces. What? Why aren't your recipients currently in awe at the glory of your amazing message?
+So, you tried to send a bunch of email, and when you checked your [Event webhook]({{root_url}}/API_Reference/Webhooks/event.html) or your [Email Activity]({{root_url}}/User_Guide/Delivery_Metrics/email_activity.html) page, you see a number of messages are marked as "Deferred" instead of "Delivered". Or maybe you're looking at your [Email Reports]({{root_url}}/User_Guide/Delivery_Metrics/email_reports.html), and noticing a lot of addresses in your [Blocks]({{site.app_url}}/suppressions/blocks), but not many in your Bounces. What? Why aren't your recipients currently in awe at the glory of your amazing message?
 
-First of all, rest assured that our system is designed with throttling (also known as "deferral" these terms mean the same thing in the email world, and may be used interchangeably) in mind. We continue to attempt to send messages for up to 72 hours until the message is delivered. Normally, it takes between a few minutes to a few hours until the message is delivered.
+Sendgrid systems have been designed with throttling in mind. We will continue to attempt to send messages for up to 72 hours until the message is delivered. Normally, it takes between a few minutes to a few hours until a deferred message is delivered, it's all up to the receiving mail server.
```

Comment:
~~+Sendgrid systems have been designed with throttling in mind. We will continue to attempt to send messages for up to 72 hours until the message is delivered. Normally, it takes between a few minutes to a few hours until a deferred message is delivered, it's all up to the receiving mail server.~~

I'm not sure what the first sentence really means, so I've removed it:

SendGrid continues to attempt to send messages for up to 72 hours or until successful delivery. Normally, it takes between a few minutes and a few hours for a deferred messages successful delivery - it depends on the receiving mail server.